internal/pkg/cli/svc_logs.go

```diff
@@ -179,6 +195,14 @@ func (o *svcLogsOpts) Execute() error {
 	if o.limit != 0 {
 		limit = aws.Int64(int64(o.limit))
 	}
+	if o.previous {
+		taskID, err := o.setPreviousStoppedTaskID()
+		if taskID != "" {
+			o.taskIDs = []string{taskID}
```

Comment:
It's odd that `setPreviousStoppedTaskID` doesn't set but returns the last stopped task ID, maybe:
```suggestion
		taskID, err := o.latestStoppedTaskID()
		if taskID != "" {
			o.taskIDs = []string{taskID}
```